src/mach/load_command.rs

```diff
@@ -1540,7 +1598,25 @@ impl<'a> ctx::TryFromCtx<'a, Endian> for CommandVariant {
                 let comm = bytes.pread_with::<LinkeditDataCommand>(0, le)?;
                 Ok((LinkerOptimizationHint(comm), size))
             }
-            _ => Ok((Unimplemented(lc), size)),
+            LC_VERSION_MIN_TVOS => {
+                let comm = bytes.pread_with::<VersionMinCommand>(0, le)?;
+                Ok((VersionMinTvos(comm), size))
+            }
+            LC_VERSION_MIN_WATCHOS => {
+                let comm = bytes.pread_with::<VersionMinCommand>(0, le)?;
+                Ok((VersionMinWatchos(comm), size))
+            }
+            LC_DYLD_EXPORTS_TRIE => {
+                let comm = bytes.pread_with::<LinkeditDataCommand>(0, le)?;
+                Ok((DyldExportsTrie(comm), size))
+            }
+            LC_DYLD_CHAINED_FIXUPS => {
+                let comm = bytes.pread_with::<LinkeditDataCommand>(0, le)?;
```

Comment:
This and above are both LinkEditDataCommand? Interesting

Reply:
Yep, they both reside in `__linkedit`, like a few others.